lib/Gitlab/Api/Issues.php

```diff
@@ -352,6 +354,26 @@ protected function createOptionsResolver()
             ->setAllowedTypes('assignee_id', 'integer')
         ;
 
+        $datetimeNormalizer = function (Options $resolver, \DateTimeInterface $value) {
+            return $value->format('c');
+        };
+
+        $resolver->setDefined('created_after')
+            ->setAllowedTypes('created_after', \DateTimeInterface::class)
```

Comment:
Please update the phpdoc of all associated methods so `created_after` is described as `string|\DateTimeInterface`, and similarly for the other cases.

Reply:
I think there may be a few other places in the codebase that also need addressing.